fapi-client/src/main/scala/com/gu/facia/api/models/curatedcontent.scala

```diff
@@ -61,7 +61,7 @@ object FaciaContent {
 
     CuratedContent(
       content,
-      trailMetaData.headline.orElse(contentFields.get("headline")),
```

Comment:
I have asked @maxharlow about this; it seems the fields work like this:
`fields.headline` is for the headline within the article, where `webTitle` is the headline for the link name on the front. We happen to have out `CuratedContent` called `headline`, so I think this is probably correct. Maybe we should change `CuratedContent.headline` to `webTitle`?